docs/en/pid-eaa-data-model.rst

```diff
@@ -149,20 +156,24 @@ The ``record`` MUST have at least the following sub parameters:
     - **Description**
     - **Reference**
   * - **type** 
-    - identification of the national eID framework used by the User. For example ``eidas.it.cie`` means that the CIE id identification scheme is used by the User. 
+    - Identification of the trust framework used for obtaining the verified claims. For example, in case of PID, ``eidas.it.cie`` means that the CIE id identification scheme is used by the User. 
     - `[OID.IDA. Section 5.1.1.2] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#section-5.1.1.2>`_
   * - **source**
     - JSON Object cointaining the follwoing mandatory claims:
 
-      - **organization_name**: Name of the Organization handling the eID used by the User
-      - **organization_id**: Identification code for the Organization. It MUST be set to the *IPA Code* of the Organization
+      - **organization_name**: Name of the Organization. In case of PID, it is the Organization handling the eID used by the User. For the (Q)EAA it is the Authentic Source.
```

Comment:
```suggestion
      - **organization_name**: Name of the Organization. In the case of PID, it is the issuer of the digital credential. For the (Q)EAAs it is the Authentic Source.
```

Reply:
This is not true. In the case of PID, it is the IdP (CIE/SPID).

Reply:
I'm still not convinced by your suggestion. The organization name here is referred to the authentic source. The organization name of the Issuer is in the EC of the Issuer. we have the iss parameter in the SD-JWT, which refers to the Issuer identifier.

Reply:
yes, I was lost and my suggestion was not related to that piece related to IDA

however I'd not write here the specific constraints about pid and (q)eaa issuance, here we should reduce the text by saying that it contains the autentic source without any IF

Reply:
Yes I agree with you, I will push a commit asap.

Reply:
@peppelinux I made a commit, please check if it is fine with you.